public/js/reader.js

```diff
@@ -331,7 +374,36 @@ Reader.updateMetadata = function () {
                 },
             });
         } else { $(".file-info").text(`${filename} :: ${width} x ${height} :: ${size} KB`); }
-    } else { $(".file-info").text(`Double-Page View :: ${width} x ${height}`); }
+    } else { 
+        // HEAD request to get filesize
+        let size = Reader.preloadedSizes[Reader.currentPage];
+        let size_pre = Reader.preloadedSizes[Reader.currentPage+1];
+        let size_view = size + size_pre;
+        if (!size || !size_pre) {
+            $.ajax({
+                url: Reader.pages[Reader.currentPage],
+                type: "HEAD",
+                success: (data, textStatus, request) => {
+                    size = parseInt(request.getResponseHeader("Content-Length") / 1024, 10);
+                    Reader.preloadedSizes[Reader.currentPage] = size;
+                    $.ajax({
+                        url: Reader.pages[Reader.currentPage+1],
+                        type: "HEAD",
+                        success: (data, textStatus, request) => {
+                            size_pre = parseInt(request.getResponseHeader("Content-Length") / 1024, 10);
+                            Reader.preloadedSizes[Reader.currentPage+1] = size_pre;
+                            size_view = size + size_pre;
+                            $(".file-info").text(`${filename} - ${filename_pre} :: ${width_view} x ${height} :: ${size_view} KB`);
+                            $(".file-info").attr("title", `${filename} :: ${width} x ${height} :: ${size} KB - ${filename_pre} :: ${width_pre} x ${height_pre} :: ${size_pre} KB`);
```

Comment:
Nice touch, but the `attr` parameter isn't removed if we switch back to single page mode.  

![image](https://user-images.githubusercontent.com/8237712/161639728-ab588e27-b21f-4d48-9541-fe084eb158f4.png)  

I think nobody will know this exists here either -- It might be interesting to move this to a small `<i>` element alongside the text, which is only shown when doublePage mode is enabled.  
(crude mockup below)
![image](https://user-images.githubusercontent.com/8237712/161640183-e5affc64-c7a8-4892-aeb7-8f86cbd4786b.png)

```
<span>
<div class="file-info" style="display: inline;">002.jpg - 003.jpg :: 2128 x 1544 :: 548 KB</div>
<i class="fa fa-info-circle" id="doublepage-detail" style="display: inline;margin-left: 8px;" title="002.jpg :: 1064 x 1544 :: 279 KB - 003.jpg :: 1064 x 1527 :: 269 KB"></i>
</span>
```